cmd/kube-apiserver/app/server.go

```diff
@@ -136,7 +137,7 @@ cluster's shared state through which all other components interact.`,
 }
 
 // Run runs the specified APIServer.  This should never exit.
-func Run(runOptions *options.ServerRunOptions, stopCh <-chan struct{}) error {
+func Run(runOptions *preparedServerRunOptions, stopCh <-chan struct{}) error {
```

Comment:
as we wrap a pointer, we can pass a value of the wrapper, can't we? Not sure how we do the same for the config.

Reply:
Yes, we can pass a value. That has not any diff in this case.

Reply:
yeah, just a nit. not a big deal.

Reply:
done